### ISSUE TITLE
--extend-ignore to extend existing extend-ignore in config

### ISSUE THREAD

**Issue by hans-d** [PyCQA/flake8#1804]
### describe the request

I need Flake8 to extend-ignore the existing extend-ignore (as per the configuration file) from the command line, so I can ad-hoc selectively disable some additional checks without remembering/looking up the existing ignored tests or changing the config.

Currently, the cli option replaces the configured one, which is expected (and documented as such). Having something like `--extend-configured-ignore` would be appreciated.

This allows for eg, doing initial linting while dev-ing focused on the code without being bothered about docstrings (which should bother me before I commit / by default).

**Comment by sigmavirus24** [comment]
In general we avoid adding any additional complexity to the CLI options and configuration parsing. As such, I'm going to decline this because you haven't completely explained what problems you're encountering (with example configuration) and even if you did, I think we would still reject the feature request

**Comment by bergentroll** [comment]
We have a project `setup.cfg` with `extend-ignore` specified and would like run flake8 in a CI pipeline with additional ignores. `--extend-ignore` overrides `extend-ignore`, defined in `setup.cfg`. Expected and option to extend existing list. `+/-` in a beginning of list or every item seems useful option. Other option would be to have `--extend-ignore` bool flag to not override ignore list, but just as an idea. Of course there is the API already.

**Comment by asottile** [comment]
why would you want to have CI be different than local?

**Comment by bergentroll** [comment]
@asottile, to skip some issues which are in progress and should not to be skipped locally.

**Comment by sigmavirus24** [comment]
Yeah this just makes me frustrated with myself for not coming up with a baseline concept and allowing folks to say "ignore these known things in the baseline but enforce rules otherwise for anything new".

**Comment by sigmavirus24** [comment]
Anyway, I get where you're coming from @bergentroll but that's not currently possible and the way you've suggested to fix it isn't a way I would ever agree to fixing it

**Comment by asottile** [comment]
make two config files is how we did this
